src/common/components/MySecurity/Crypto/Crypto.style.ts

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
```

Comment:
As discussed in the group, we will use `TailwindCSS` instead of `styled-components` so please use `TailwindCSS` and remove `styled-components`

Reply:
Fixed